### ISSUE TITLE
It's urgent, why is it that my training and reasoning directly using the source code is so slow?

### ISSUE THREAD

**Issue by txw-github** [lyuwenyu/RT-DETR#296]
I'm using the pytorch version on an RTX 3060 and it takes 4-5min for a training dataset of 6,000 sheets and 45s for a test set of 2,000 sheets. whereas yolov8 only takes 1.3min and 17s

**Comment by txw-github** [comment]
I'm looking for guidance, I need it urgently.
